packages/dialtone-icons/package.json

```diff
@@ -6,18 +6,17 @@
     "build": "gulp"
   },
   "files": [
-    "dist"
+    "dist",
+    "vue2/dist",
+    "vue3/dist"
   ],
   "exports": {
-    "./vue2": {
-      "import": "./dist/dialtone-icon@2.js",
-      "require": "./dist/dialtone-icon@2.cjs"
-    },
-    "./vue3": {
-      "import": "./dist/dialtone-icon@3.js",
-      "require": "./dist/dialtone-icon@3.cjs"
-    },
-    "./*": "./dist/*"
+    "./vue2": "./vue2/dist/dialtone-icons.js",
+    "./vue3": "./vue3/dist/dialtone-icons.js",
+    "./vue2/*": "./vue2/dist/components/*.js",
+    "./vue3/*": "./vue3/dist/components/*.js",
```

Comment:
Hmm so does this mean consumers have to import components like `import DtIcon from '@dialpad/dialtone/vue2/icon'` or the code splitting doesn't work? Maybe will require testing product side, could depend on how the bundler is configured?

Reply:
It's not enforced, consumers can still import from `'@dialpad/dialtone/vue2'`.

Importing from `DtIcon from '@dialpad/dialtone/vue2/icon'` instead of `{ DtIcon } from '@dialpad/dialtone/vue2'` it's a way to enforce the tree-shake behavior regardless of the consumer bundler config.

Reply:
Okay sweet that's what I was hoping.